libcentrifugo/application.go

```diff
@@ -498,7 +498,7 @@ func (app *Application) pubPing() error {
 		Goroutines: runtime.NumGoroutine(),
 		NumCPU:     runtime.NumCPU(),
 		Gomaxprocs: runtime.GOMAXPROCS(-1),
-		Metrics:    *app.metrics,
+		Metrics:    *app.metrics.GetSnapshot(),
```

Comment:
Don't understand why this happened exactly too, but one of races I posted in comments looks like panic trace - https://gist.github.com/FZambia/88bc50efec3728ea7f85 - so hope this was an actual panic reason